docs/Contributing/API-for-contributors.md

```diff
@@ -1345,18 +1345,18 @@ If the `name` is not already associated with an existing team, this API route cr
         "macos_settings": {
           "custom_settings": {
             "path": "path/to/profile1.mobileconfig"
-            "labels": ["Label 1", "Label 2"]
+            "labels_include_all": ["Label 1", "Label 2"]
           },
           {
             "path": "path/to/profile2.json"
-            "labels": ["Label 3", "Label 4"]
+            "labels_exclude_any": ["Label 3", "Label 4"]
           },
```

Comment:
We should update this documentation. This is invalid JSON. I think custom_settings is meant to be an array here

Reply:
It is! Good catch. Thanks Gabe.

Reply:
Please feel free to tweak the docs if it still doesn't look right after my latest commit (cc2a3f2)

Reply:
`custom_settings` should be an array of objects. I updated this.

Reply:
Thank you!